sitestats/sitestats-tool/src/java/org/sakaiproject/sitestats/tool/wicket/pages/ReportsEditPage.java

```diff
@@ -913,13 +916,16 @@ public String getStyle(Object object) {
 				SelectOption opt = (SelectOption) object;
 				ToolModel toolModel = (ToolModel) opt.getDefaultModel();
 				String toolId = toolModel.getToolId();
-				if(!ReportManager.WHAT_EVENTS_ALLTOOLS.equals(toolId)) {
-					String toolIconPath = "background-image: url(" + Locator.getFacade().getEventRegistryService().getToolIcon(toolId) + ");";
-					String style = "background-position:left center; background-repeat:no-repeat; margin-left:3px; padding-left:20px; "+toolIconPath;
-					return style;
-				}
-				return null;
-			}		
+				String style = "display:block;";
+				return style;
+			}
+			public String getClass(Object object) {
```

Comment:
the use of method names getClass should probably be avoided

Reply:
Changed to getIconClass.